packages/purpose-process/src/model/domain/models.ts

```diff
@@ -0,0 +1,45 @@
+import { DelegationId, EServiceMode } from "pagopa-interop-models";
+import { z } from "zod";
+
+export const ownership = {
+  CONSUMER: "CONSUMER",
+  PRODUCER: "PRODUCER",
+  SELF_CONSUMER: "SELF_CONSUMER",
+} as const;
+export const Ownership = z.enum([
+  Object.values(ownership)[0],
+  ...Object.values(ownership).slice(1),
+]);
+export type Ownership = z.infer<typeof Ownership>;
+
+export const PurposeDocumentEServiceInfo = z.object({
+  name: z.string(),
+  mode: EServiceMode,
+  producerName: z.string(),
+  producerIPACode: z.string().optional(),
+  consumerName: z.string(),
+  consumerIPACode: z.string().optional(),
```

Comment:
Align casing with the other model
```suggestion
  consumerIpaCode: z.string().optional(),
```